Makefile

```diff
@@ -1,89 +1,45 @@
 export GO15VENDOREXPERIMENT := 1
 
-UNAME := $(shell uname)
-ifeq ($(UNAME), Darwin)
-    HASH := md5
-else
-    HASH := md5sum
-endif
-
 all: build manifests
 
-generate: sync
-	find pkg/ -name "*generated*.go" -exec rm {} -f \;
-	./hack/build-go.sh generate ${WHAT}
-	goimports -w -local kubevirt.io cmd/ pkg/ tests/
-	./hack/bootstrap-ginkgo.sh
-	(cd tools/openapispec/ && go build)
-	tools/openapispec/openapispec --dump-api-spec-path api/openapi-spec/swagger.json
+generate:
+	hack/dockerized hack/glide-checksync.sh && ./hack/generate.sh
 
-apidocs: generate
-	./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html
+apidocs:
+	hack/dockerized hack/glide-checksync.sh && ./hack/generate.sh && ./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html
 
-build: checksync fmt vet compile
+build:
+	hack/dockerized hack/glide-checksync.sh && ./hack/check.sh && ./hack/build-go.sh install ${WHAT}
 
 goveralls:
-	./hack/goveralls.sh
-
-compile:
-	./hack/build-go.sh install ${WHAT}
-
-vet:
-	./hack/build-go.sh vet ${WHAT}
-
-fmt:
-	goimports -w -local kubevirt.io cmd/ pkg/ tests/
+	hack/dockerized ./hack/goveralls.sh
 
-fmt-bash:
-	shfmt -i 4 -w cluster/ hack/ images/
-
-test: build
-	./hack/build-go.sh test ${WHAT}
+test:
+	hack/dockerized ./hack/build-go.sh test ${WHAT}
 
 functest:
-	./hack/functests.sh
+	hack/functests.sh
 
 clean:
-	./hack/build-go.sh clean ${WHAT}
-	rm _out/ -rf
-	rm tools/openapispec/openapispec -rf
+	hack/dockerized "./hack/build-go.sh clean ${WHAT} && rm _out/* -rf && rm tools/openapispec/openapispec -rf"
 
 distclean: clean
-	find manifests -name "*.yaml" -type f -delete
-	rm -rf vendor/
-	rm -f .glide.*.hash
-	glide cc
+	hack/dockerized "rm -rf vendor/ && rm -f .glide.*.hash && glide cc"
```

Comment:
I'm not sure doing distclean within the docker volume has the desired effect? Similar issue with clean above; could you outline the dist process given the docker building?

Reply:
right. :+1:

Reply:
Done.